src/dry-command-interceptor.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import Process = NodeJS.Process;
+import { Cli } from './cli';
+import { NpmCommandProxy } from './npm-command-proxy';
+
+/**
+ * Responsible for the requested dry specific command interception
+ */
+export class DryCommandInterceptor {
+    private readonly rawArgs: string[];
+    private npmCommandProxy: NpmCommandProxy;
+    private drySpecificArgs: string[] = ['saveTo'];
+    /**
+     * @param {Cli} cli The CLI to use
+     * @param {NodeJS.Process} process The main process
+     */
+    constructor(cli: Cli, process: Process) {
+        this.rawArgs = process.argv.slice(2);
+        this.npmCommandProxy = new NpmCommandProxy(cli, this.rawArgs);
```

Comment:
We should avoid to call Npm with dry specific arguments.
NpmCommandProxy constructor should not take `rawArgs` as a parameter, instead `rawArgs` should be passed as a parameter to the proxy method.

Reply:
Ok done. Dry arguments are removed from the arguments given to the proxyfied command